charts/pg-operator/values.yaml

```diff
@@ -1,140 +1,45 @@
----
-# ======================
-# Installer Controls
-# ======================
+# Default values for pg-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+operatorImageRepository: percona/percona-postgresql-operator
+imagePullPolicy: IfNotPresent
+image: ""
+
+# set if you want to specify a namespace to watch
+# defaults to `.Release.namespace` if left blank
+# watchNamespace:
+
+# set if operator should be deployed in cluster wide mode. defaults to false
+watchAllNamespaces: false
+
+# set if you want to use a different operator name
```

Comment:
**issue**: the `operatorName` value doesn't appear to be used by the chart, the comment references `xtradb`
**suggestion**: remove this section from the values file and add comments for the `nameOverride` and `fullnameOverride` fields